CHANGELOG.md

```diff
@@ -4,6 +4,18 @@
 
 [1]: https://www.npmjs.com/package/@google-cloud/security-center?activeTab=versions
 
+## v0.3.0
+
+03-14-2019 07:55 PDT
+
+***This release has breaking changes!**.  The `having` enum value from `GroupFindingsRequest` has been removed.  This field was not supposed to be exposed.
```

Comment:
I think there were some formatting errors here.

```diff
- ***This
+ **This

- breaking changes!**.
+ breaking changes!**
```

Reply:
Woah, NICE CATCH!

Reply:
Woo! One last thing is that period that comes after the **. I would add a commit to change it, but I don't know if dealing with the CLA bot is more work for you than if you just edit the file yourself!

Reply:
don't know where my head was with this one....

Reply:
aha! more arguments for automation :stuck_out_tongue: